CHANGELOG.md

```diff
@@ -14,6 +14,7 @@
 - HiDPI support for Wayland.
 - `EventsLoop::get_available_monitors` and `EventsLoop::get_primary_monitor` now have identical counterparts on `Window`, so this information can be acquired without an `EventsLoop` borrow.
 - `AvailableMonitorsIter` now implements `Debug`.
+- In X11, all events loops will now share the same XConnection.
```

Comment:
"events loops" -> "event loops" (I know that's the actual type name, but it's a very awkward type name).

The other entries don't use the future tense.

Need backticks around `XConnection`.